### DISCUSSION TITLE
Allow to filter by team on filters like author

### DISCUSSION THREAD

**Discussion by hackaugusto** [community/community#20587] [Pull Requests]
ATM to find out all the PRs of my team mates I have to construct a search like `is:pr author:@handle1 author:@handle2`, it would be nice to support `is:pr author:org/our-team` to simplify creating/maintaining such a search.

**Comment by ghost** [comment]
> ATM to find out all the PRs of my team mates I have to construct a search like `is:pr author:@handle1 author:@handle2`, it would be nice to support `is:pr author:org/our-team` to simplify creating/maintaining such a search.
